### DISCUSSION TITLE
Error 409 questions

### DISCUSSION THREAD

**Discussion by KZ29-hub** [microsoft/BioGPT#113] [General]
When I want to download Pre-trained BioGPT model by using  wget https://msramllasc.blob.core.windows.net/modelrelease/BioGPT/checkpoints/Pre-trained-BioGPT.tgz. But I encountered errors: error 409：Public access is not permitted on this storage account.。 Could I ask how can I solve this problem?

**Comment by eclausen667** [comment]
did you figure out a way around this?

**Reply by eclausen667** [reply]
@renqianluo would you happen to know if this is intentional that the checkpoints are no longer public?
